### PR TITLE
[EDU-2305] feat: add object storage API reference pt

### PR DESCRIPTION
**Pull request by gabriel-azion**
Related issue:

[EDU-2305](https://aziontech.atlassian.net/browse/EDU-2305)

Changes:

- add object storage API reference

[EDU-2305]: https://aziontech.atlassian.net/browse/EDU-2305?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by bru-andrade — APPROVED** [review]
Nice! 👯

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 1]
```diff
@@ -0,0 +1,377 @@
+---
+title: Object Storage API 
+description: A Object Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
+meta_tags: edge function, edge computing, storage, jamstack, apis
+namespace: documentation_runtime_storage
+permalink: /documentacao/devtools/runtime/api-reference/storage/
+menu_namespace: runtimeMenu
+---
+
+A Object Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:10]:
```suggestion
A  [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/) Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 2]
```diff
@@ -0,0 +1,377 @@
+---
+title: Object Storage API 
+description: A Object Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:3]:
```suggestion
description: A Edge Functions Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+
+```js
+async Storage.put(key, value, options)
+```
+
+#### Parâmetros
+
+| Parâmetro | Tipo             | Descrição                                                                                             |
+| --------- | --------------- | ----------------------------------------------------------------------------------------------------- |
+| `key`     | string          | Identificador que permite a busca por um objeto no storage.                                     |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:62]:
```suggestion
| `key` | String  | Identificador que permite a busca por um objeto no storage |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+| `value`   | ArrayBuffer ou ReadableStream | Conteúdo do objeto sendo armazenado. No caso de `value` implementar um [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), as APIs de Stream podem ser usadas e a opção `content-length` é obrigatória. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:63]:
```suggestion
| `value` | ArrayBuffer ou ReadableStream | Conteúdo do objeto sendo armazenado. No caso de `value` implementar um [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream), as APIs de Stream podem ser usadas e a opção `content-length` é obrigatória |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 5]
```diff
@@ -0,0 +1,377 @@
...
+| `options` | object          | Os atributos `options` são descritos na tabela abaixo.                                                 |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:64]:
```suggestion
| `options` | Object | Os atributos `options` são descritos na tabela abaixo |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 6]
```diff
@@ -0,0 +1,377 @@
...
+
+**options**
+
+Para passar o parâmetro `options`, você deve fornecer um objeto com os seguintes atributos:
+
+| Atributo          | Tipo   | Descrição                                                                                                                                                        |
+| ----------------- | ------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `content-type`    | string | O `content-type` do objeto sendo criado. É semelhante ao cabeçalho HTTP `content-type` e descreve como o conteúdo do Object Storage deve ser tratado. `content-type` é opcional; o valor padrão é `application/octet-stream`. |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:72]:
```suggestion
| `content-type` | String | O `content-type` do objeto sendo criado. É semelhante ao cabeçalho HTTP `content-type` e descreve como o conteúdo do Object Storage deve ser tratado. `content-type` é opcional. O valor padrão é `application/octet-stream` |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 7]
```diff
@@ -0,0 +1,377 @@
...
+| `content-length`  | string | O tamanho do objeto a ser criado, em bytes. É obrigatório quando o valor é ReadableStream.                                                                       |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:73]:
```suggestion
| `content-length` | String | O tamanho do objeto a ser criado, em bytes. É obrigatório quando o valor é ReadableStream |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 8]
```diff
@@ -0,0 +1,377 @@
...
+| `metadata`        | object | Qualquer objeto JavaScript contendo informações sobre o objeto que será armazenado no Storage. Todas as propriedades do objeto devem ser strings.              |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:74]:
```suggestion
| `metadata` | Object | Qualquer objeto JavaScript contendo informações sobre o objeto que será armazenado no Storage. Todas as propriedades do objeto devem ser strings |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 9]
```diff
@@ -0,0 +1,377 @@
...
+
+```js
+async storage.get(key)
+```
+
+#### Parâmetros
+
+| Parâmetro | Tipo   | Descrição                                                                                          |
+| --------- | ------ | -------------------------------------------------------------------------------------------------- |
+| `key`     | string | Identificador que permite a busca por um objeto no bucket.                                          |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:187]:
```suggestion
| `key` | String | Identificador que permite a busca por um objeto no bucket |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 10]
```diff
@@ -0,0 +1,377 @@
...
+
+```js
+async Storage.delete(key)
+```
+
+#### Parâmetros
+
+| Parâmetro | Tipo   | Descrição                                                                                          |
+| --------- | ------ | -------------------------------------------------------------------------------------------------- |
+| `key`     | string | Identificador que permite a busca por um objeto no bucket.                                          |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:286]:
```suggestion
| `key` | String | Identificador que permite a busca por um objeto no bucket  |
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 11]
```diff
@@ -0,0 +1,377 @@
...
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ |
+| `StorageObject.content`                     | Propriedade de leitura contendo um ReadableStream para o conteúdo armazenado no storage.               |
+| `async StorageObject.arrayBuffer()`         | Função assíncrona que retorna um ArrayBuffer com o conteúdo armazenado no storage. Este método consome o ReadableStream da propriedade de content. |
+| `StorageObject.metadata`                    | Propriedade de leitura contendo um [objeto Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) com os metadados do objeto armazenados no storage. |
+| `StorageObject.contentType`                 | Propriedade de leitura contendo o `content-type` do conteúdo armazenado no storage.                    |
+| `StorageObject.contentLength`               | Propriedade de leitura contendo o tamanho em bytes do conteúdo armazenado no storage.                 |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:375]:
```suggestion
| `StorageObject.content` | Propriedade de leitura contendo um ReadableStream para o conteúdo armazenado no storage |
| `async StorageObject.arrayBuffer()` | Função assíncrona que retorna um ArrayBuffer com o conteúdo armazenado no storage. Este método consome o ReadableStream da propriedade de content |
| `StorageObject.metadata` | Propriedade de leitura contendo um [objeto Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) com os metadados do objeto armazenados no storage |
| `StorageObject.contentType` | Propriedade de leitura contendo o `content-type` do conteúdo armazenado no storage |
| `StorageObject.contentLength` | Propriedade de leitura contendo o tamanho em bytes do conteúdo armazenado no storage |
```

**Review by MarianaAguilera — APPROVED** [review]

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 12]
```diff
@@ -0,0 +1,381 @@
...
+description: A Edge Functions Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
+meta_tags: edge function, edge computing, storage, jamstack, apis
+namespace: documentation_runtime_storage
+permalink: /documentacao/devtools/runtime/api-reference/storage/
+menu_namespace: runtimeMenu
+---
+
+A [Edge Functions](/pt-br/documentacao/produtos/edge-application/edge-functions/) Storage API é uma interface que permite o acesso ao Azion Object Storage. Através dessa interface, é possível ler e escrever dados no storage e seus buckets.
+
+## Instanciação da classe
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:12]:
```suggestion

---

## Instanciação da classe
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx [hunk 13]
```diff
@@ -0,0 +1,383 @@
...
+
+### Valor de Retorno
+
+Objeto `Storage` usado para acessar o Azion Object Storage.
+
+---
+
+## Métodos
+
+### `async storage.put`
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/api-reference/object-storage/object-storage.mdx:50]:
```suggestion
### `async Storage.put`
```

**Pull request merged**